### PR TITLE
Fix 475: Incorrect content in client screen, if data is not available

### PR DESCRIPTION
**Pull request by mayank-kgp**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@therajanmaurya Please have a look at this PR.

**Comment by mayank-kgp** [comment]
@therajanmaurya Waiting for your review.  :)

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java [hunk 1]
```diff
@@ -552,7 +568,19 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i,
 
     @Override
     public void showFetchingError(String s) {
-        Toast.makeText(getActivity(), s, Toast.LENGTH_SHORT).show();
+        if (!Network.isOnline(getContext())) {
```
Comment (droidchef) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: `View` should not contain business logic. This should be controlled with the presenter.

**Comment by droidchef** [comment]
Needs improvement.

**Pull request closed without merging**